lib/index.js

```diff
@@ -396,7 +396,7 @@ const Popover = createClass({
       !this.containerEl.contains(event.target) &&
       !this.targetEl.contains(event.target)
     )
-    if (isOuterAction) this.props.onOuterAction()
+    if (isOuterAction) this.props.onOuterAction(event)
```

Comment:
:+1: Thanks! Nitpick on the next PR you make (hopefully there is a next one! : ) try to keep this good but unrelated improvements out of the PR. It makes reviewing proposals easier for me and it also makes changelog links more readable.